pkg/metrics/metrics.go

```diff
@@ -1247,6 +1254,17 @@ func CreateConfiguration(metricsEnabled []string) (Configuration, []prometheus.C
 
 			collectors = append(collectors, ArpingRequestsTotal)
 			c.ArpingRequestsTotalEnabled = true
+
+		case Namespace + "_endpoint_propagation_delay_seconds":
+			EndpointPropagationDelay = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+				Namespace: Namespace,
+				Name:      "endpoint_propagation_delay_seconds",
+				Help:      "CiliumEndpoint propagation delay in seconds",
```

Comment:
maybe "CiliumEndpoint roundtrip propagation delay in seconds"? Without roundtrip this could be interpreted as propagation delay between nodes if one just reads this help msg.

Reply:
Done.